src/LightSaml/Context/AbstractContext.php

```diff
@@ -154,7 +154,7 @@ public function clearSubContexts()
     /**
      * @return \ArrayIterator
      */
-    public function getIterator()
+    public function getIterator(): \Traversable
```

Comment:
ArrayIterator does not implémente Traversable (https://www.php.net/manual/fr/class.arrayiterator.php)
Can you use the good type and remove the phpdoc ?